src/main/java/uk/gov/companieshouse/documentsigningapi/environment/EnvironmentVariablesChecker.java

```diff
@@ -16,7 +16,6 @@ public enum RequiredEnvironmentVariables {
         AWS_REGION("AWS_REGION"),
         AWS_ACCESS_KEY_ID("AWS_ACCESS_KEY_ID"),
         AWS_SECRET_ACCESS_KEY("AWS_SECRET_ACCESS_KEY"),
-        AWS_SESSION_TOKEN("AWS_SESSION_TOKEN"),
```

Comment:
Is this removed environment variable present in the README? If so it can be removed there also.

Reply:
Good spot! I will update the README too.